Framework/API/src/SpectrumInfo.cpp

```diff
@@ -132,6 +132,14 @@ Kernel::V3D SpectrumInfo::position(const size_t index) const {
   return newPos / static_cast<double>(dets.size());
 }
 
+/// Returns the phi angle (in radians) for the spectrum with given index
+double SpectrumInfo::phi(const size_t index) const {
```

Comment:
This method is deliberately not part of `SpectrumInfo`. The problem is that its naming and functionality is ambiguous, confusing, and misleading. `Detector::getPhi` returns the spherical component (angle around Z), but there are several algorithms the interpret it as the angle around the beam direction. For many instruments this is ok, since the beam is aligned with Z, but in general it is not. Those algorithm will thus give wrong results.

We are considering to later add a method to obtain the angle around the beam direction but it will definitely not be named `phi()`.

Please remove this method.